### DISCUSSION TITLE
代理协议选择

### DISCUSSION THREAD

**Discussion by kulongwangzhi85** [v2fly/v2ray-core#2012] [Q&A]
https://www.v2fly.org/v5/config/proxy/vless.html

提示： VLESS 已被弃用并且可能被移除。

传输流还想用quic，该如何选择代理协议？

**Comment by CrazyBoyFeng** [comment]
可以用 trojan 代替，页面上有提示。
配置方法跟 vless 差不多，把 `uuid` 换成 `password` 即可。

vless 被弃用其实也并不意味着它有明显问题，只是因为这个协议没人维护了。
从原理上讲，trojan 跟 vless 没多大差别，都是相当简单的协议。不带加密混淆，首包认证后明文传输。
